pallets/api/src/fungibles/tests.rs

```diff
@@ -100,14 +183,25 @@ fn increase_allowance_works() {
 		let owner = ALICE;
 		let spender = BOB;
 
-		create_token_and_mint_to(ALICE, token, ALICE, value);
-		assert_eq!(0, Assets::allowance(token, &ALICE, &BOB));
-		assert_ok!(Fungibles::increase_allowance(signed(ALICE), token, BOB, value));
-		assert_eq!(Assets::allowance(token, &ALICE, &BOB), value);
+		// Check error works for `Assets::approve_transfer()`.
+		assert_noop!(
+			Fungibles::increase_allowance(signed(owner), token, spender, value),
+			AssetsError::Unknown.with_weight(AssetsWeightInfo::approve_transfer())
+		);
+		pallet_assets_create_and_mint_to(owner, token, owner, value);
+		for origin in vec![root(), none()] {
+			assert_noop!(
+				Fungibles::increase_allowance(origin, token, spender, value),
+				BadOrigin.with_weight(WeightInfo::approve(0, 0))
+			);
+		}
+		assert_eq!(0, Assets::allowance(token, &owner, &spender));
```

Comment:
this uses `(expected, actual)` when the other tests use `(actual, expected)`

Reply:
+1